kanopi/rooseveltedu
# Adding Paragraph Types

## Description
Paragraph Types

## Related Tickets

* [25888413: Paragraphs](https://kanopi.teamwork.com/#/tasks/25888413)

## Steps to Validate

1. Navigate to [Paragraph Types](http://ci-315-rooseveltedu.pantheonsite.io/admin/structure/paragraphs_type).

2.Click **Manage fields** for each paragraph type and verify that they have the correct fields using the [blueprint(https://docs.google.com/spreadsheets/d/1F6Y8rOghRSQqAGEcleseYnzEosZ01MN43eSndj--PWg/edit#gid=1862302892)
